src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -989,19 +989,28 @@ namespace webnn_native { namespace ie {
         } else {
             attrs.shape_calculation_mode = ngraph_shape_calc_mode::Scales;
         }
-        const ngraph_node_t* axes =
-            AddConstantWithGraph<int64_t>(precision_e::I64, {4}, {0, 1, 2, 3});
-        ngraph_node_t* resampleNode;
-        if (transpose) {
+        // axes: Interpolate layer only supports resize on spatial dimensions(depth, height and
+        // width)
+        // https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/nodes/mkldnn_interpolate_node.cpp#L2515
+        const ngraph_node_t* axesNode =
+            AddConstantWithGraph<int32_t>(precision_e::I32, {2}, {2, 3});
```

Comment:
@fujunwei  As marked by the spec of OpenVINO, only 4D input tensor with 'axes = {2, 3}' is supported for "mode" == "linear_onnx". But we only used 'nearest' and 'linear' modes. When I set axes to {0, 1} or other values, IE will throw an error.